test/library/cluster_operator.go

```
@@ -0,0 +1,37 @@
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	configclient "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+)
+
+// WaitForClusterOperatorAvailableNotProgressingNotDegraded waits for a ClusterOperator to report
+// Available=True, Progressing=False, and Degraded=False status conditions.
+func WaitForClusterOperatorAvailableNotProgressingNotDegraded(ctx context.Context, client configclient.ConfigV1Interface, name string) error {
+	return wait.PollImmediate(5*time.Second, 5*time.Minute, func() (bool, error) {
```

golangci-lint (2.11.4)

[error] SA1019: wait.PollImmediate is deprecated: This method does not return errors from context, use PollUntilContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release. 

(staticcheck)